frontends/election-manager/src/lib/backends/storage_backend.ts

```diff
@@ -426,4 +431,27 @@ export class ElectionManagerStoreStorageBackend
       return parsedElectionDefinition;
     }
   }
+
+  setWriteIns(writeIns?: readonly Admin.WriteInRecord[]): Promise<void> {
+    if (writeIns) {
```

Comment:
this code should never be run right? so why do we need to also implement this?

Reply:
Correct, but I wanted it for validation purposes. It'll be removed eventually.

Reply:
got it, definitely feels like a good reason to finish the refactor sooner rather then later as it feels like a lot of overhead to have to pointlessly implement everything here too. I just put in the function but threw a not implemented error to get around the validation issue in my branch.

Reply:
To be clear, I meant _validation of the concept_, not type checking.